### DISCUSSION TITLE
[Feature Request] Ability to set custom image save location in settings

### DISCUSSION THREAD

**Discussion by ArchAngelAries** [BlafKing/sd-civitai-browser-plus#100] [Ideas]
Would love a feature for the extension where I can set the save location for images when saving using the "Save Images" button in the settings instead of manually pasting the folder path each time. Would help keep model folders tidy and prevent accidental saving of models in wrong folders, which has happened when I just assumed the path would auto change back when selecting a different model.

**Comment by BlafKing** [comment]
Sure thing! I'll add that as a setting in the next update.
I'll add an extra setting where you can define a path which will then override the download folder location when saving images! :)
Sidenote, it should reset the download folder each time when you click a new model though, From my testing it returns the folder that the model is installed at each time you click a model.

**Comment by BlafKing** [comment]
This has been implemented since v1.15.2

You can set this option in the settings tab of CivitAI Browser+! :)
